spec/rspec/rails/matchers/be_valid_spec.rb

```diff
@@ -22,6 +22,9 @@ class Boat
     def valid?
       false
     end
+
+    def errors(_)
+    end
```

Comment:
We need a seperate spec and class for this, as this removes coverage of an edgecase